src/SFA.DAS.ApprenticeAccounts.UnitTests/DTOs/ApprenticeSync/WhenMappingFromApprenticeToApprenticeSyncDto.cs

```diff
@@ -0,0 +1,51 @@
+﻿using NUnit.Framework;
+using SFA.DAS.ApprenticeAccounts.Data.Models;
+using SFA.DAS.Testing.AutoFixture;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System;
+using AutoFixture.NUnit3;
+using System.Net.Mail;
+using SFA.DAS.ApprenticeAccounts.DTOs.Apprentice;
+using FluentAssertions;
+
+namespace SFA.DAS.ApprenticeAccounts.UnitTests.DTOs.ApprenticeSync;
+public class WhenMappingFromApprenticeToApprenticeSyncDto
+{
+    [Test]
+    [RecursiveMoqAutoData]
+    public void ThenTheFieldsAreMappedCorrectly(
+        [Frozen] Guid apprenticeId,
+        [Frozen] string firstName,
+        [Frozen] string surname,
+        [Frozen] MailAddress email,
+        [Frozen] DateTime dateOfBirth,
+        [Frozen] DateTime updatedSinceDate
```

Comment:
Use of Frozen is inappropriate here. Instead, you could have only one parameter here `Apprentice apprentice`